annotations/annotations_rw.go

```diff
@@ -0,0 +1,96 @@
+package annotations
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	tidUtils "github.com/Financial-Times/transactionid-utils-go"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+)
+
+type RW interface {
```

Comment:
I know that there are no write operations at the moment. I put `RW` in the prospective of having the same interface implementing the write functionalities, since the app will interact with the same microservice for writing.

I did not name the interface `AnnotationsRW` because it will appear as `annotations.AnnotationsRW` outside the package. I think that `annotations.RW` is more concise and readable.